docker-entrypoint.sh

```diff
@@ -131,6 +131,15 @@ if [ "$1" = 'rabbitmq-server' ]; then
 		export RABBITMQ_CTL_ERL_ARGS="$RABBITMQ_SERVER_ADDITIONAL_ERL_ARGS"
 	fi
 
+	echo
+	for f in /docker-entrypoint-initdb.d/*; do
+		case "$f" in
```

Comment:
Do you mean `-x` for testing executable status?

The `*.sh` case is sourcing the script, not just executing it, so it can actually affect the surrounding environment.